Cargo.toml

```diff
@@ -2,13 +2,12 @@
 name = "mls-assist"
 version = "0.1.0"
 edition = "2021"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+LICENSE = "GPL-3.0"
```

Comment:
This should be AGPL

Reply:
No, it's AGPL like all other repositories: https://github.com/phnx-im/mls-assist/blob/main/LICENSE. Am I missing something?